packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -154,6 +154,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 className={ css('ms-BasePicker-input', styles.pickerInput) }
                 ref={ this._resolveRef('input') }
                 onFocus={ this.onInputFocus }
+                onBlur={ this.onInputBlur }
```

Comment:
Since onBlur is already one of inputProps you don't need to add it here, although it makes things a bit more consistent which is nice.